Fabric.Authorization.API/Modules/RootModule.cs

```diff
@@ -8,6 +8,7 @@ public class RootModule : NancyModule
         public RootModule(): base("/v1")
         {
             Get("/", _ => Redirect(), null, "Redirect");
```

Comment:
Can we use the `AccessControl.Path` constant here?

Reply:
Good call, I'll update that.